atlascope/settings.py

```diff
@@ -25,6 +26,18 @@ class AtlascopeMixin(ConfigMixin):
         'analytics',
     ]
 
+    # Use PostGIS
+    DATABASES = values.DatabaseURLValue(
+        environ_name='DATABASE_URL',
+        # django-configurations has environ_prefix=None by default here
+        environ_prefix='DJANGO',
+        environ_required=True,
+        # Additional kwargs to DatabaseURLValue are passed to dj-database-url,
+        # then passed through to the Django database options.
+        engine='django.contrib.gis.db.backends.postgis',
+        conn_max_age=600,
+    )
+
```

Comment:
Can we make this change more precise with `mutate_configuration`?

Reply:
I filed an issue upstream to see if this can be simplified: https://github.com/girder/django-composed-configuration/issues/161.